src/WebJobs.Script/Workers/ProcessManagement/WorkerProcess.cs

```diff
@@ -216,10 +216,10 @@ public void Dispose()
                 {
                     if (!Process.HasExited)
                     {
-                        Process.Kill();
                         if (!Process.WaitForExit(processExitTimeoutInMilliseconds))
                         {
-                            _workerProcessLogger.LogWarning($"Worker process has not exited despite waiting for {processExitTimeoutInMilliseconds} ms");
+                            Process.Kill();
+                            _workerProcessLogger.LogInformation($"Worker process has not exited despite waiting for {processExitTimeoutInMilliseconds} ms");
```

Comment:
Changing log level from _warning_ to _info_ otherwise for all the workers not handling WorkerTerminate, we will see warnings in logs. Not a hard preference though from my end, I can switch it back to warning if it is needed for something else.
@fabiocav, @brettsam let me know your thoughts.

Reply:
Looks like this comment is no longer relevant as the code has changed? But I vote for warning still :)

Reply:
Yup this not valid now. Thanks